### DISCUSSION TITLE
SB Lern - Release 5.0

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Lern/Community#13] [Documentation]
The details of the items planned for Release 5.0 of SB Lern have been published on the Lern microsite.
The 'Product Roadmap' page has the list of the items : https://lern.sunbird.org/learn/product-roadmap
The team will use this thread to call out any updates/ changes in plan to release 5.0

**Comment by reshmi-nair** [comment]
Release notes for SB Lern Release-5.0 is published in the microsite. Release is live with the planned scope. [https://lern.sunbird.org/use/release-notes-1/release-v-5.0.0-latest](https://lern.sunbird.org/use/release-notes-1/release-v-5.0.0-latest)

_Note: CSP related changes were added to the 5.0 scope later on and are not complete yet. Once it is completed release notes will be updated._
